crates/opencascade-sys/cpp/wrapper.cpp

```diff
@@ -149,6 +149,10 @@ std::unique_ptr<BRepAlgoAPI_Fuse> BRepAlgoAPI_Fuse_ctor(const TopoDS_Shape& shap
   return std::unique_ptr<BRepAlgoAPI_Fuse>(new BRepAlgoAPI_Fuse(shape_1, shape_2));
 }
 
+std::unique_ptr<BRepAlgoAPI_Cut> BRepAlgoAPI_Cut_ctor(const TopoDS_Shape& shape_1, const TopoDS_Shape& shape_2){
+    return std::unique_ptr<BRepAlgoAPI_Cut>(new BRepAlgoAPI_Cut(shape_1, shape_2));
+}
```

Comment:
Sorry I don't have an automated code formatting set up yet for C++ code, but could you format this in the same style as the existing code?

I'll try to get auto code formatting and CI checks later.

Reply:
Fixed.